ci-operator/config/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-main.yaml

```diff
@@ -3,6 +3,14 @@ base_images:
     name: "4.10"
     namespace: ocp
     tag: cli
+  golangci-lint:
```

Comment:
Where we will use golangci-lint container

Reply:
removing this